NetduinoDeploy/NetduinoDeploy/NetduinoDeploy/Managers/OtpManager.cs

```diff
@@ -165,10 +171,20 @@ bool WriteOtp(byte[] data)
             var devices = DfuContext.Current.GetDevices();
             DfuDevice device = devices[0];
 
+            device.Uploading += Device_Uploading;
+
             device.Upload(data, 0x1FFF7800, 2);
+
+            device.Uploading -= Device_Uploading;
             return true;
         }
 
+        void Device_Uploading(object sender, UploadingEventArgs e)
```

Comment:
is this needed?

Reply:
Oh yes, I forgot to raise the event .... I'll get that fixed :)